apps/central-scan/frontend/src/components/export_results_modal.tsx

```diff
@@ -32,43 +30,31 @@ enum ModalState {
   INIT = 'init',
 }
 
-export function ExportResultsModal({
-  onClose,
-  numberOfBallots,
-}: Props): JSX.Element {
+export function ExportResultsModal({ onClose }: Props): JSX.Element {
   const [currentState, setCurrentState] = useState<ModalState>(ModalState.INIT);
   const [errorMessage, setErrorMessage] = useState('');
 
-  const { usbDriveEject, usbDriveStatus, auth, logger } =
-    useContext(AppContext);
+  const { usbDriveEject, usbDriveStatus, auth } = useContext(AppContext);
   assert(isElectionManagerAuth(auth));
   const userRole = auth.user.role;
+  const exportCastVoteRecordsToUsbDriveMutation =
+    exportCastVoteRecordsToUsbDrive.useMutation();
 
-  async function exportResults() {
+  function exportResults() {
     setCurrentState(ModalState.SAVING);
-
-    try {
-      await logger.log(LogEventId.SaveCvrInit, userRole);
-
-      await exportCastVoteRecords();
-
-      setCurrentState(ModalState.DONE);
-      await logger.log(LogEventId.SaveCvrComplete, userRole, {
-        message: `Successfully saved CVR file with ${numberOfBallots} ballots.`,
-        disposition: 'success',
-        numberOfBallots,
-      });
```

Comment:
I'm revisiting logging holistically in a follow-up PR (leaning toward consolidating logging in app backends)